### PR TITLE
Add Chew keyboard

### PR DESCRIPTION
**Pull request by flinguenheld**
## Description

Hello,
I've created a new 34 keys keyboard with a RP2040 Zéro.
You can see the repo with all information [here](https://github.com/flinguenheld/chew).

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/chew/config.h [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+/* disable debug print */
+//#define NO_DEBUG
+
+/* disable print */
+//#define NO_PRINT
+
+/* disable action features */
+//#define NO_ACTION_LAYER
+//#define NO_ACTION_TAPPING
+//#define NO_ACTION_ONESHOT
```
Comment (waffle87) [inline keyboards/chew/config.h:28]:
```suggestion
```

File: keyboards/chew/info.json
Comment (waffle87) [inline keyboards/chew/info.json:1]: Please run `qmk format-json` on this file.

File: keyboards/chew/keymaps/default/config.h
Comment (waffle87) [inline keyboards/chew/keymaps/default/config.h:1]: This file should be removed.

File: keyboards/chew/info.json [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+                    ["GP29", "GP28", "GP27", "GP26", "GP15"],
+                    ["GP8", "GP9", "GP13", "GP14", "NO_PIN"],
+                    ["GP5", "GP6", "GP7", "NO_PIN", "NO_PIN"]
+                ]
+            }
+        }
+    },
+
+    "layouts": {
+        "LAYOUT_split_5_5_4_3": {
```
Comment (waffle87) [inline keyboards/chew/info.json:53]:
```suggestion
        "LAYOUT": {
```

File: keyboards/chew/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      * ├───┼───┼───┼───┼───┘       └───┼───┼───┼───┼───┤
+      * │ Z │ X │ C │ V │               │ M │ , │ . │ / │
+      * └───┴───┴───┴───┘               └───┴───┴───┴───┘
+      *           ┌───┐                   ┌───┐
+      *           │ B ├───┐           ┌───┤ N │
+      *           └───┤Bsp├───┐   ┌───┤Ent├───┘
+      *               └───┤Alt│   │   ├───┘
+      *                   └───┘   └───┘
+      */
+    [0] = LAYOUT_split_5_5_4_3(
```
Comment (waffle87) [inline keyboards/chew/keymaps/default/keymap.c:23]:
```suggestion
    [0] = LAYOUT(
```

File: keyboards/chew/keymaps/default/rules.mk
Comment (waffle87) [inline keyboards/chew/keymaps/default/rules.mk:1]: This file should be removed. This does not belong at the keymap level.

File: keyboards/chew/readme.md
Comment (waffle87) [inline keyboards/chew/readme.md:1]: All images here need to switched to the standard markdown format of
```markdown
![alt](link)
```

File: keyboards/chew/readme.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+- Read the wiki to have some help or information: [Chew wiki](https://github.com/flinguenheld/chew/wiki)
+
+<div align="center">
+  <img width="200px" src="https://github.com/flinguenheld/chew/blob/main/images/squirrel_brown.png?raw=true">
+</div>
+
+### Requirements
+
+- 2x PCB
+- 2x MCU board [RP2040-Zéro](https://www.waveshare.com/wiki/RP2040-Zero)
```
Comment (waffle87) [inline keyboards/chew/readme.md:21]:
```suggestion
- 2x MCU board [RP2040-Zero](https://www.waveshare.com/wiki/RP2040-Zero)
```

File: keyboards/chew/rules.mk [hunk 5]
```diff
@@ -0,0 +1 @@
+# This file intentionally left blank
```
Comment (waffle87) [inline keyboards/chew/rules.mk:1]:
```suggestion
SERIAL_DRIVER = vendor
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/chew/info.json [hunk 6]
```diff
@@ -0,0 +1,87 @@
+{
+    "manufacturer": "florent@linguenheld.fr",
+    "keyboard_name": "chew",
+    "maintainer": "florent@linguenheld.fr",
+    "bootloader": "rp2040",
+    "features": {
+        "bootmagic": false,
```
Comment (waffle87) [inline keyboards/chew/info.json:7]:
```suggestion
        "bootmagic": true,
```

**Review by waffle87 — APPROVED** [review]
Otherwise, lgtm.

File: keyboards/chew/info.json [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+                "direct": [
+                    ["GP0", "GP1", "GP2", "GP3", "GP4"],
+                    ["GP29", "GP28", "GP27", "GP26", "GP15"],
+                    ["GP8", "GP9", "GP13", "GP14", "NO_PIN"],
+                    ["GP5", "GP6", "GP7", "NO_PIN", "NO_PIN"]
+                ]
+            }
+        },
+        "transport": {
+            "protocol": "serial_usart",
```
Comment (zvecr) [inline keyboards/chew/info.json:36]: ` "protocol": "serial_usart"` doesnt do anything.

```suggestion
```

File: keyboards/chew/info.json [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+            "sync": {
+                "matrix_state": true
+            }
```
Comment (zvecr) [inline keyboards/chew/info.json:39]: Syncing matrix state with no rgb/oled is somewhat pointless.

```suggestion
            "watchdog": true
```

File: keyboards/chew/config.h [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+#pragma once
+
+/* Flash */
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET              // Activates the double-tap behavior
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_TIMEOUT 200U // In ms in which the double tap can occur
+
+#define EE_HANDS
+#define SERIAL_USART_TX_PIN GP11
+#define SERIAL_PIO_USE_PIO11
+#define SPLIT_WATCHDOG_ENABLE
```
Comment (zvecr) [inline keyboards/chew/config.h:13]:
```suggestion
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**